src/lib/fragments/AgGrid.react.js

```diff
@@ -386,9 +417,20 @@ export default class DashAgGrid extends Component {
     }
 
     componentDidMount() {
+        const {id} = this.props;
+        if (id) {
+            agGridRefs[stringifyId(id)] = this.reference.current;
```

Comment:
`this.props.id` is guaranteed to be a string already; so this one, and the ones in `componentWillUnmount` and `componentDidUpdate` don't need `stringifyId`, it's only reading `alignedGrids` where we need this.